dhctl/pkg/kubernetes/actions/resources/resources.go

```diff
@@ -36,7 +36,7 @@ import (
 	"github.com/deckhouse/deckhouse/dhctl/pkg/util/retry"
 )
 
-var ErrNotAllResourcesCreated = fmt.Errorf("Not all resources were creatated")
+var ErrNotAllResourcesCreated = fmt.Errorf("not all resources were created")
```

Comment:
Please keep upper case here. We using uppercase everywhere because errors will display to users